### DISCUSSION TITLE
soft fail after setting up ip pools

### DISCUSSION THREAD

**Discussion by The-3x** [postalserver/postal#1908] [Help with using Postal]
I've created ip pool with 2 different ips with valid hostnames.
https://prnt.sc/peUEDuRmWg9A

Also i have the ips in my network inteface. (ping works fine, telnet with 25 working fine)

Also i have added A records for the hostnames and pointed them to ips and created rdns record. but when i try to send email using smtp it doesn't allow me to send and all mails finishes with a SOFT FAIL.
https://prnt.sc/T2oL70nE7VpV

What can i do?

**Comment by willpower232** [comment]
> no smtp servers were available

this suggests your server provider is blocking your connections on email ports, can you confirm with them whether they are?

**Reply by masihfathi** [reply]
hi i have the same issue with new docker install and ip pool. all settings seems to be correct. pool ips can't be access

**Reply by The-3x** [reply]
hi, i have fixed the issue. i am giving you some solutions you can try.
1. you must need to add A record and TXT record for each ip hostnames. for my case node1.xxxxxx.com & also include all your ip pools hostname in the spf record so sending from other domains won't make it looks spammy. example: https://prnt.sc/1eMvZQphXRSL
2. you should add proper RDNS record for each ips which will be used as HELO. because google and other providers block mails if you don't have matching RDNS & HELO

**Comment by PrabhuShanmughapriyan** [comment]
hi.. What is the purpose of hostname in IP pools.

if **hostname1** has **(IP 1 , IP 2)** and **hostname2** has **IP 3,**  if a postal worker running on **hostname1** when it sends email will pick **IP1 or IP2**  ?
